compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -483,6 +484,17 @@ impl<'hir> Map<'hir> {
         }
     }
 
+    pub fn visit_exported_macros_in_krate<V>(&self, visitor: &mut V)
+    where
+        V: Visitor<'hir>,
+    {
+        let krate = self.krate();
+
+        for id in krate.exported_macros {
```

Comment:
```suggestion
        for id in self.krate().exported_macros {
```

Reply:
Good!
I missed the part because I made it by looking at the above function!